pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -734,220 +734,224 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 
   app.controller('SectionAssessmentsController', ['$scope', '$routeParams', '$window', '$q', '$timeout', '$interval', '$sanitize', 'sectionsService', 'studentsService',
                                              function($scope, $routeParams, $window, $q, $timeout, $interval, $sanitize, sectionsService, studentsService) {
+    // Some strings.
+    var submission_list = {
```

Comment:
Should all this javascript be in it's own file? I didn't even realize this was a haml file. Not familiar with angular though.